BigCommerce/Integration/04_ManageProductData.md

```diff
@@ -0,0 +1,99 @@
+
+3. Verify that the image at the first position is suitable for your *BigCommerce* store. If not, drag and drop the desired image to the first position.
+
+4. Verify that the *Alt text* field is filled for the image on first position. If you must change the existing entry, you have the following options: 
+    - You can change the entry. This might have impact on other connections you have in use.
+    - If and want to keep the product data in this scope unchanged, copy your products to a specific *BigCommerce* sales channel. <!---Stimmt das, sollte man das so tun?-->
+
+5. If applicable, click the [SAVE] button to save your changes.
+
+
+
+## Configure color&dash;swatch variant option
+
+Configure the color-swatch variant option for the *BigCommerce* products for which the options provided in the connection settings are not suitable. 
+A color-swatch option in *BigCommerce* is an option with which the store customers can select a color combination in the *BigCommerce* storefront. For example, you want to offer T-shirts that are available with the color combinations black/white, white/green, and black/green. *BigCommerce* supports up to three colors per color swatch value.   
+ For detailed information on the standard selection options, see [Configure BigCommerce connection](../Integration/01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).
+
+#### Prerequisites
+
+- To add a variant option as a color-swatch, the defining attribute in the variant set needs to be of the *Tree node* data type.
+- If you have the *PIM* module not in use, you can check or create the attribute in the *DataHub* module.
+
+#### Procedure
+
+*PIM > Settings > Tab Attributes*
+
+ ![PIM attributes](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/PIMattribute.png "[PIM attributes]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Create attribute* view is displayed.
+
+    ![Create attribute](../../Assets/Screenshots/PIM/Settings/Attributes/CreateAttribute.png "[Create attribute]")
+
+2. Enter a name for the attribute in the *Name* field and, if desired, add an attribute description in the *Description* field.
+
+3. Select the *Tree node* data type in the *Data type* drop-down list.  
+   The ![Add](../../Assets/Icons/Plus05.png "[Add]") (Add) button in the *Assigned sets* field is unlocked. The *CONFIGURATION* section is displayed.    
+
+4. Ensure, that the *Multi-language* and *Multi-channel* toggles are disabled.
+
+    > [Info] Be aware that only attributes which are neither multi-language nor multi-channel can be used as defining attributes for variants. For detailed information, see [Create a variant set](../../PIM/Integration/07_ManageVariantSets.md#create-a-variant-set).
+
+5. Click the ![Add](../../Assets/Icons/Plus05.png "[Add]") (Add) button in the *Assigned sets* field. The button is locked if you have not yet selected a data type.   
```

Comment:
The button is locked if you have not yet selected a data type. --> Diese Info trifft hier nicht wirklich zu (Data type ist im Schritt 3 ausgewählt worden). Evtl. hier weglassen?

Reply:
aus anderer Doku kopiert...